### PR TITLE
Added examples and fixed other errors in swagger spec

### PR DESCRIPTION
**Pull request by prakharsharma10**
In this PR, we have addressed all the validation errors in the swagger spec. The changes include:

1. Added x-ms-examples for all the operations.
2. Removed 'location' and 'tags' properties from 'CustomDomain' and 'EdgeNode' as we don't have these in our RP.
3. Added terminal response codes for long running operations.
4. Made 'tag' as an additional property for the 'Profiles_Update' operation.

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by anuchandy** [comment]
instead of adding `id, type & name` properties to edgeNode & customDomain.. Define following models and do allOf on `ProxyResource`. For the top level resources do allOf on `TrackedResource`. You can refer traffic manager spec for [example](https://github.com/Azure/azure-rest-api-specs/blob/current/specification/trafficmanager/resource-manager/Microsoft.Network/2017-05-01/trafficmanager.json).


```json
    "Resource": {
      "description": "The core properties of ARM resources",
      "properties": {
        "id": {
          "readOnly": true,
          "type": "string",
          "description": "Fully qualified resource Id for the resource. Ex - /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/trafficManagerProfiles/{resourceName}"
        },
        "name": {
          "readOnly": true,
          "type": "string",
          "description": "The name of the resource"
        },
        "type": {
          "readOnly": true,
          "type": "string",
          "description": "The type of the resource. Ex- Microsoft.Network/trafficmanagerProfiles."
        }
      },
      "x-ms-azure-resource": true
    },
    "TrackedResource": {
      "description": "The resource model definition for a ARM tracked top level resource",
      "properties": {
        "tags": {
          "type": "object",
          "additionalProperties": {
            "type": "string"
          },
          "x-ms-mutability": [
            "read",
            "create",
            "update"
          ],
          "description": "Resource tags."
        },
        "location": {
          "type": "string",
          "x-ms-mutability": [
            "read",
            "create"
          ],
          "description": "The Azure Region where the resource lives"
        }
      },
      "allOf": [
        {
          "$ref": "#/definitions/Resource"
        }
      ]
    },
    "ProxyResource": {
      "description": "The resource model definition for a ARM proxy resource. It will have everything other than required location and tags",
      "allOf": [
        {
          "$ref": "#/definitions/Resource"
        }
      ]
    },
```

File: specification/cdn/resource-manager/readme.md [hunk 1]
```diff
@@ -85,77 +85,3 @@ csharp:
   output-folder: $(csharp-sdks-folder)/Cdn/Management.Cdn/Generated
   clear-output-folder: true
 ```
-
```
Comment (anuchandy) [inline specification/cdn/resource-manager/readme.md]: Why following entires are removed this is needed in readme file.

**Review by anuchandy — CHANGES_REQUESTED** [review]

**Comment by prakharsharma10** [comment]
Thanks @anuchandy!  Addressed both comments.

**Comment by anuchandy** [comment]
@prakharsharma10 thank you!. Could you pls take a look at the model validation errors reported by [CI](https://travis-ci.org/Azure/azure-rest-api-specs/builds/312718584?utm_source=github_status&utm_medium=notification) here is the [link to validation errors](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/312718592)

**Comment by prakharsharma10** [comment]
@anuchandy The errors are mostly about adding examples for all the response codes while in this PR, we have added examples for each operation as we're only targeting the linter errors to meet the swagger violations KPI. We'll add the examples for all response codes in the next set of PRs. Thanks!

**Comment by anuchandy** [comment]
@prakharsharma10 sounds good! thanks for cleaning up the errors. To track model validation issue just have created a work item [here](https://github.com/Azure/azure-rest-api-specs/issues/2116). Merging this PR.

**Review by anuchandy — APPROVED** [review]

**Pull request merged**